src/containers/Dashboard/Search/component.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import {
+  Row, Col, Input
+} from 'antd'
+import PropTypes from 'prop-types'
+import { Form, Field } from 'formik'
+import InputField from 'Components/InputField'
+import ClearButton from './ClearButton'
+
+const Search = ({
+  handleSubmit, values, onClear, handleBlur
+}) => (
+  <Row
+    justify="center"
+    gutter={{
+      xs: 8, sm: 16, md: 24, lg: 22
+    }}
+  >
+    <Col
+      className="gutter-row"
+      xs={{ span: 20 }}
+      sm={{ span: 20 }}
+      md={{ span: 14 }}
+      lg={{ span: 12 }}
+      xl={{ span: 10 }}
+    >
+      <Form>
+        <Field
+          component={InputField}
+          inputAs={Input.Search}
+          name="search"
+          placeholder="Enter movie name"
+          size="large"
+          enterButton="Search"
+          className="top-margin"
+          onSearch={handleSubmit}
+          onBlur={values.search && handleBlur}
```

Comment:
why do we need this?

Reply:
to validate input by blur only if it has the value

Reply:
do you need this behavior only for this field?

Reply:
@VladWithCookies yes